app/models/form_profiles/va526ez.rb

```diff
@@ -156,4 +167,29 @@ def prefill_military_address(address)
       address_line_3: address&.address_three
     }.compact
   end
+
+  def initialize_payment_information(user)
+    return {} unless user.authorize :evss, :access?
+
+    service = EVSS::PPIU::Service.new(user)
+    response = service.get_payment_information
+    raw_account = response.responses.first&.payment_account
+
+    if raw_account
+      VA526ez::FormPaymentAccountInformation.new(
+        account_type: raw_account&.account_type&.capitalize,
+        account_number: mask(raw_account&.account_number),
+        routing_number: mask(raw_account&.financial_institution_routing_number),
+        bank_name: raw_account&.financial_institution_name
+      )
+    else
+      {}
+    end
+  rescue StandardError
+    {}
```

Comment:
do we want add a `Rails.logger.error` here so we'd know what was up if it started always returning `{}`?

Reply:
Good call, considering recent events having more logging wont hurt.